README.md

```diff
@@ -1,37 +1,59 @@
+**WARNING:** Uniqys Kit is still in beta version, and we may still make some breaking changes.
+
+Uniqys Kit is a framework for building Decentralized Applications (or DApps).
+It is designed for web developers to easily build DApps: just as you always build your web app.
+
+For details, visit our [project website](https://uniqys.net/kit) and [documentation](https://uniqys.github.io/UniqysKitDocs/).
+
+Also, feedbacks are appreciated via GitHub issues, [Gitter](https://gitter.im/uniqys/UniqysKit), or [Twitter](https://twitter.com/uniqys).
+
+## Introduction
+
+Uniqys Kit has two main components:
+
+- Chain Core: Builds and manages a blockchain on a Tendermint consensus algorithm.
+- Easy Framework: A framework for developers to create DApps without being aware of the blockchain.
 
-- Chain Core provides fast and scalable blockchain infrastructure by using the sidechain technology.
-- Easy Framework enables developers to create DApps without being conscious of the blockchain.
+These two packages were designed to remove some of the current obstacles in creating or using DApps.
 
-For details, please visit our [website](https://uniqys.net/kit) and [documentation](https://uniqys.github.io/UniqysKitDocs/).
+### How Does It Work?
 
-## Current status
+Please check our [documentation](https://uniqys.github.io/UniqysKitDocs/).
 
-This is a beta version of Uniqys Kit, and it is still under development.
-It is ready for building a blockchain and is able to take consensus to maintain a sole application state in the network.
+### TL;DR
 
-We would appreciate if you could give us your feedback via GitHub issue, [Gitter](https://gitter.im/uniqys/UniqysKit-preview), or [Twitter](https://twitter.com/uniqys).
+- Chain Core establishes an application-friendly blockchain network.
+  - **Scalable:** Creates a single blockchain for every single DApp.
+  - **Fast:** Instant finality and high transaction throughput are obtained with the Tendermint alogorithm.
+  - **Flexible:** Developers can decide the most suitable transaction fees and incentives to the network maintainer.
+- Easy Framework achieves "DApp development ≒ Web app development".
+  - Building a REST API and a frontend is all you need to do to develop a DApp.
+  - No programming language restriction.
+  - Data storing on blockchain will be as simple as using a Memcached protocol.
+  - Sending a transaction is sending a POST request.
+- Track external events (i.e. Ethereum Events) in the blockchain network so that it can become a side chain.
 
-### Chain Core
+## Getting Started
 
-`chain-core-dev` package is removed, and `chain-core` package is available.
-`chain-core` implements Tendermint consensus algorithm and thereby it can construct a secure blockchain network.
-It only supports static validator set, and dynamic validator set will be implemented in the future version.
+### Installing via npm
 
-### Easy Framework
+```sh
+$ npm install @uniqys/cli
```

Comment:
Cloud you add `-g` option.
https://uniqys.github.io/UniqysKitDocs/introduction/install.html#node-js

Reply:
fixed